Remotion/Web/Development.WebTesting/WebDriver/Factories/Remote/RemoteBrowserFactory.cs

```diff
@@ -0,0 +1,126 @@
+﻿// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using Coypu;
+using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Edge;
+using OpenQA.Selenium.Firefox;
+using OpenQA.Selenium.Remote;
+using Remotion.Web.Development.WebTesting.BrowserSession;
+using Remotion.Web.Development.WebTesting.BrowserSession.Remote;
+using Remotion.Web.Development.WebTesting.Configuration;
+using Remotion.Web.Development.WebTesting.WebDriver.Configuration.Remote;
+
+namespace Remotion.Web.Development.WebTesting.WebDriver.Factories.Remote;
+
+/// <summary>
+/// Factory to create a browser session using Selenium remote driver.
+/// </summary>
+public class RemoteBrowserFactory : IBrowserFactory
```

Comment:
how much of this is duplicated from the orignal browsers? should we leave comments? is there a risk about projects needing to duplicate browser options?

Reply:
we should find a way to de-duplicate this extension point or make it clear also for projects. Follow-up ticket?